web/modules/custom/feedback/src/Form/FeedbackForm.php

```diff
@@ -1,187 +0,0 @@
-<?php
-/**
- * @file
- * Contains \Drupal\feedback\Form\FeedbackForm
- */
-namespace Drupal\feedback\Form;
-
-use Drupal\Core\Config\ConfigFactoryInterface;
-use Drupal\Core\Database\Connection;
-use Drupal\Core\Form\FormBase;
-use Drupal\Core\Form\FormStateInterface;
-use Drupal\Core\Session\AccountProxyInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
-use Symfony\Component\HttpFoundation\RequestStack;
-
-/**
- * Provides an feedback form.
- */
-class FeedbackForm extends FormBase {
-
-  /**
-   * The Database Connection.
-   *
-   * @var \Drupal\Core\Database\Connection
-   */
-  protected $database;
-
-  /**
-   * @var \Symfony\Component\HttpFoundation\RequestStack
-   */
-  protected $requestStack;
-
-  /**
-   * @var \Drupal\Core\Session\AccountProxyInterface
-   */
-  protected $currentUser;
-
-  /**
-   * @var \Drupal\Core\Config\ConfigFactoryInterface
-   */
-  protected $configFactory;
-
-  /**
-   * FeedbackForm constructor.
-   *
-   * @param \Drupal\Core\Database\Connection $database
-   *
-   * @param \Symfony\Component\HttpFoundation\RequestStack $request_stack
-   *
-   * @param \Drupal\Core\Session\AccountProxyInterface $current_user
-   *
-   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
-   *
-   */
-  public function __construct(Connection $database, RequestStack $request_stack, AccountProxyInterface $current_user, ConfigFactoryInterface $config_factory) {
-    $this->database = $database;
-    $this->requestStack = $request_stack;
-    $this->currentUser = $current_user;
-    $this->configFactory = $config_factory;
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public static function create(ContainerInterface $container) {
-    return new static(
-      $container->get('database'),
-      $container->get('request_stack'),
-      $container->get('current_user'),
-      $container->get('config.factory')
-    );
-  }
-
-  /**
-   * (@inheritdoc)
-   */
-  public function getFormId() {
-    return 'feedback_form';
-  }
-  /**
-   * (@inheritdoc)
-   */
-  public function buildForm(array $form, FormStateInterface $form_state) {
-    $getFilterValue = $this->requestStack->getCurrentRequest()->query->get('num');
```

Comment:
what is 'num'?

code is not obvious. use more human-friendly names and add comments